rwt/check_package.py

```diff
@@ -0,0 +1,76 @@
+import pip
+from contextlib import redirect_stdout
+import os
+import re
+"""
+This helps use the methods from pkg-config (github.com/matze/pkgconfig) without having to make changes to environment
```

Comment:
I double-checked the license on matze/pkgconfig and it's the same MIT license, so that's not a problem.